### PR TITLE
Bug/89 blog page missing the label of form input

### PR DESCRIPTION
**Pull request by Cameron933**
### Descriptions
1. Added label for input in subscription section
2. Adjusted InfoBlock css for text line breaks

- Link to Zenhub Card: https://app.zenhub.com/workspaces/taodemy-63c3dec267d0df73ca6087cb/issues/gh/taodemy/website/89
- Link to UI design (if possible): no need

### Provide screenshots or videos that the function is working.
image
image
image

### Provide screenshots of no warning, or error in the console.
image

- [x] All unit tests passing and coverage of testing is more than 90% unless special needs.
- [x] No merge conflict with the `main` branch, make sure your code is up to date with `main`.
- [x] Check PR title is aligned with your ticket.
- [x] Check your app will run without crashes.
- [x] Check if there are hard-coded values.
- [x] No `//eslint-disable-next-line no-unused-vars`, unless otherwise specified.
- [x] Check you are using `camelCase && PascalCase` correctly.
- [x] Remove unused comments.

### CODE REVIEW

File: components/general/InfoBlock/InfoBlock.tsx [hunk 1]
```diff
@@ -56,11 +56,15 @@ const InfoBlock = ({
...
           <form className={styles.info_block__sign_up_content__form}>
-            <input
-              type="email"
-              defaultValue="YOUR EMAIL"
-              className={`global__heading-h4 ${fontSyne.className} ${styles.info_block__sign_up_content__form__input}`}
-            />
+            <div className={styles.info_block__sign_up_content__form__group}>
+              <label htmlFor="email" className={`global__heading-h4 ${fontSyne.className}`}>
+                YOUR EMAIL
+              </label>
```
Comment (HarryuCATGREY) [inline components/general/InfoBlock/InfoBlock.tsx:62]: One possible solution for keep label exist and hide it is put it outside the user view like follows:
image

**Review by HarryuCATGREY — CHANGES_REQUESTED** [review]

**Comment by Cameron933** [comment]
> image
> 
> image
> 
> I think it's different with the figma

image
image

**Review by HarryuCATGREY — APPROVED** [review]
LGTM

**Review by jiayuu-u — APPROVED** [review]
LGTM

**Pull request merged**